test/WebSites/LocalizationWebSite/Controllers/HomeController.cs

```diff
@@ -2,15 +2,28 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Localization;
 
 namespace LocalizationWebSite.Controllers
 {
     public class HomeController : Controller
     {
-        // GET: /<controller>/
+        private readonly IHtmlLocalizer _localizer;
+
+        public HomeController(IHtmlLocalizer<HomeController> localizer)
+        {
+            _localizer = localizer;
+        }
+
         public IActionResult Index()
         {
             return View();
         }
+
+        public IActionResult LocPage()
+        {
+            ViewBag.Message = _localizer["Learn More"];
```

Comment:
Avoid `ViewBag` in functional tests; use `ViewData` instead since it performs significantly better and we want to encourage its use.